pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -169,3 +169,12 @@ def init_optimizers(self, trainer: "Trainer", model: LightningModule):
 
     def optimizer_step(self, optimizer: torch.optim.Optimizer, lambda_closure: Callable, **kwargs):
         optimizer.step(closure=lambda_closure, **kwargs)
+
+    @property
+    def setup_optimizers_after_dispatch(self) -> bool:
+        """
+        Override to delay setting optimizers and schedulers till after dispatch.
+        This is useful when the `TrainingTypePlugin` requires operating on the wrapped accelerator model.
+        Returns: True if delaying setup optimizers till after dispatch, False to call within setup.
+        """
+        return False
```

Comment:
do we want such property we even moved from it with model optimization...

Reply:
I'm unsure what you mean here

Reply:
I am quite sure you shall not be here :D go away from the computer...

Reply:
ok, it seems I was wrong and we still have it
https://github.com/PyTorchLightning/pytorch-lightning/blob/2708c3993dfc66989884d8e2971eba7036fcbd82/pytorch_lightning/core/lightning.py#L166